eth_tester/utils/backend_testing.py

```diff
@@ -315,6 +315,37 @@ def test_send_transaction(self, eth_tester, test_transaction):
 
         self._send_and_check_transaction(eth_tester, test_transaction, accounts[0])
 
+    @pytest.mark.parametrize(
+        'test_transaction',
+        (
+            SIMPLE_TRANSACTION,
+            TRANSACTION_WTH_NONCE,
+            CONTRACT_TRANSACTION_EMPTY_TO,
+            CONTRACT_TRANSACTION_MISSING_TO,
+        ),
+        ids=[
+            'Simple transaction',
+            'Transaction with nonce',
+            'Create Contract - empty to',
+            'Create Contract - missing to',
+        ],
+    )
+    def test_get_transaction_receipt_byzantium(self, eth_tester, test_transaction):
+        if eth_tester.backend.__class__.__name__ == 'PyEthereum16Backend':
```

Comment:
`if isinstance(eth_tester.backend, PyEthereum16Backend)`

though I believe the pattern for skipping tests on specific backends is not to do it here in the main test module but rather here: 

https://github.com/PabloLefort/eth-tester/blob/218b66eb87d542d28bfdf3546846997df9220e54/tests/backends/test_pyethereum16.py#L32-L34